### PR TITLE
add .aicoe-ci.yaml

### PR DESCRIPTION
**Pull request by bissenbay**
Fixes: https://github.com/thoth-station/invectio/issues/96

### CODE REVIEW

**Comment by bissenbay** [comment]
@fridex @harshad16 I would like you to pay closer attention to the added `.aicoe-ci.yaml` file because I just copy/pasted from `adviser` and I don't see `invectio` directory in https://github.com/thoth-station/thoth-application, so, to be honest, I'm not quiet sure whether we need add it or not?

File: .aicoe-ci.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  build-stratergy: Source
+  registry: quay.io
+  registry-org: thoth-station
+  registry-project: invectio
+  registry-secret: thoth-station-thoth-pusher-secret
+deploy:
+  project-org: "thoth-station"
+  project-name: "thoth-application"
+  image-name: "invectio"
+  overlay-contextpath: "invectio/overlays/test/imagestreamtag.yaml"
```
Comment (harshad16) [inline .aicoe-ci.yaml:16]: This is not needed as invectio is a python package module, not a application component.

File: .aicoe-ci.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+check:
+  - thoth-build
+  - thoth-precommit
+  - thoth-pytest-py38
+build:
```
Comment (harshad16) [inline .aicoe-ci.yaml:5]: Add the following code snippet instead of the build: 
```
release:
  - upload-pypi-sesheta
```

File: .aicoe-ci.yaml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+check:
+  - thoth-build
```
Comment (harshad16) [inline .aicoe-ci.yaml:2]: remove this check as well.

**Review by harshad16 — CHANGES_REQUESTED** [review]
Thanks for the changes, suggested the changes 
please have a look

**Comment by bissenbay** [comment]
thanks @harshad16 , PTAL https://github.com/thoth-station/invectio/pull/97/commits/11cc16770ae5b8d0dd03d1beabbe96839b29addf

**Comment by bissenbay** [comment]
@fridex @harshad16 PTAL

**Review by harshad16 — APPROVED** [review]
thanks :100:

**Review by fridex — APPROVED** [review]
:100:

**Pull request merged**